hexrdgui/main_window.py

```diff
@@ -5,6 +5,7 @@
 import tempfile
 
 import h5py
+from hexrdgui.median_filter_dialog import MedianFilterDialog
```

Comment:
The imports are sorted like this:

* system imports (imports included in Python's standard library, like functools, os, etc.)
* Most third-party modules (unless noted below)
* Qt modules
* hexrd modules
* hexrdgui modules

They're also all alphabetized (or they're supposed to be, at least).

Can you put this new import with the other `hexrdgui` imports, in alphabetical order?